### ISSUE TITLE
Make the user input field and button pretty

### ISSUE THREAD

**Issue by MStewGT** [MStewGT/FindMeBeer#4]
Need to add more CSS to clean up the look and feel of the input field and submit button.

**Comment by Grill3dCheese** [comment]
Hello @MStewGT - I'd be happy to work on this! Could you assign it to me?

**Comment by MStewGT** [comment]
@Grill3dCheese Sure, go ahead and play around with it. Excited to see what you come up with. I really like your portfolio site btw!

**Comment by Grill3dCheese** [comment]
Wow, @MStewGT - hearing that alone makes me feel like I'm doing something right! 😃 Thank you so much! I really appreciate your comment about my portfolio and for taking time to look over it!! I'm stuck at work (10hr shifts) today, tomorrow and Monday but I'm going to try and wiggle in as much time as I can to work on this! Just wanted to let you know why/where my delay is coming from. If I'm not able to get something made up over this weekend I'll definitely get some examples sent over on Tuesday when I'm off. I apologize for my delay.

**Comment by Grill3dCheese** [comment]
@MStewGT - Sorry again for my delay! I was able to spend some time on the input field and button today. I made three examples but can definitely modify as you'd like. Let me know your thoughts and which you like best. I used Font Awesome (free version) to get the icons - just FYI.

![inputButtons-example](https://user-images.githubusercontent.com/25601250/96653767-7abb7c80-12ff-11eb-9b30-2a065759318c.gif)
